src/gpodder/util.py

```diff
@@ -1480,14 +1480,32 @@ def gui_open(filename):
     """
     try:
         if gpodder.ui.win32:
-            os.startfile(filename)
+            opener = None
         elif gpodder.ui.osx:
-            Popen(['open', filename], close_fds=True)
+            opener = 'open'
         else:
-            Popen(['xdg-open', filename], close_fds=True)
+            opener = 'xdg-open'
+
+        if opener is None:
+            os.startfile(filename)
+        else:
+            opener_fullpath = shutil.which(opener)
+            if opener_fullpath is None:
+                raise Exception((_("System default player '%(opener)s' not found"))
```

Comment:
Should be `default program` like below, since the directory opener isn't trying to play a file.

Reply:
Done